src/Playwright.Tests/BeforeUnloadTests.cs

```diff
@@ -21,7 +21,7 @@ public async Task ShouldRunBeforeunloadIfAskedFor()
             var dialogEvent = new TaskCompletionSource<IDialog>();
             newPage.Dialog += (_, dialog) => dialogEvent.TrySetResult(dialog);
 
-            var pageClosingTask = newPage.CloseAsync(new PageCloseOptions { RunBeforeUnload = true });
+            var pageClosingTask = newPage.CloseAsync(new() { RunBeforeUnload = true });
```

Comment:
I am just wondering whether it might be too new for the users to be able to learn from our tests.

Reply:
I had that same thought initially, but then... We should encourage usage of new language features. And if users care about what's happening, they'll have a look at the method signature and see that it's options.